wapitiCore/attack/mod_ssrf.py

```diff
@@ -58,7 +58,7 @@ def mutate(self, request: Request):
         # self._attacks_per_url_pattern[request.hash_params] += estimation
 
         for params_list in [get_params, post_params, file_params]:
-            for i in range(len(params_list)):
+            for i, _ in enumerate(params_list):
```

Comment:
You should use double underscore otherwise you are overwriting the translation function